### ISSUE TITLE
test_accuracy not reported

### ISSUE THREAD

**Issue by asafmanor** [vinits5/pcrnet_pytorch#8]
In both `test_pcrnet.py` and `train_pcrnet.py`, the function `test_one_epoch` does not return the test accuracy expected.

A) It raises a `RuntimeError` as the number of expected outputs is 2.
B) The rotation error and translation error is never reported. As to rotation error - there are several ways to measure it. What would be your golden standard for measuring it (mathematical formula)?

Disclaimer - I cited PCRNet in a paper I've published, called [SampleNet](https://github.com/itailang/SampleNet).
You can check our own implementation in PyTorch (this was prior to the publication of this repository) under the **registration** folder.

Thanks,
Asaf.

**Comment by vinits5** [comment]
Hello @asafmanor, 
I am glad that you found our work helpful for your research.
I really appreciate that you pointed out the bugs and limitations in our PCRNet's implementation.

A) I have resolve the bug and now the code should run smoothly.
B) I have modified the test_pcrnet.py file to compute the rotation and translation error.

This repository is only to provide PCRNet's code in Pytorch. But, our original implementation is https://github.com/vinits5/pcrnet in tensorflow.
